specs/_features/epbs/beacon-chain.md

```diff
@@ -0,0 +1,1450 @@
+      - [`process_pending_balance_deposits`](#process_pending_balance_deposits)
+      - [Modified `process_effective_balance_updates`](#modified-process_effective_balance_updates)
+      - [Modified `process_slashings`](#modified-process_slashings)
+      - [Modified `get_unslashed_attesting_indices`](#modified-get_unslashed_attesting_indices)
+  - [Execution engine](#execution-engine)
+    - [Request data](#request-data)
+      - [New `NewInclusionListRequest`](#new-newinclusionlistrequest)
+    - [Engine APIs](#engine-apis)
+    - [New `notify_new_inclusion_list`](#new-notify_new_inclusion_list)
+  - [Block processing](#block-processing)
+    - [Modified `process_block_header`](#modified-process_block_header)
+    - [Modified `process_operations`](#modified-process_operations)
+      - [Modified Proposer slashings](#modified-proposer-slashings)
+      - [Modified Attester slashings](#modified-attester-slashings)
+      - [Modified `process_attestation`](#modified-process_attestation)
+      - [Modified `get_validator_from_deposit`](#modified-get_validator_from_deposit)
+      - [Modified `apply_deposit`](#modified-apply_deposit)
+      - [Payload Attestations](#payload-attestations)
+      - [Execution Layer Withdraw Requests](#execution-layer-withdraw-requests)
+    - [Modified `process_withdrawals`](#modified-process_withdrawals)
+    - [New `verify_execution_payload_header_envelope_signature`](#new-verify_execution_payload_header_envelope_signature)
+    - [New `process_execution_payload_header`](#new-process_execution_payload_header)
+    - [New `verify_execution_payload_signature`](#new-verify_execution_payload_signature)
+    - [New `verify_inclusion_list_summary_signature`](#new-verify_inclusion_list_summary_signature)
+    - [Modified `process_execution_payload`](#modified-process_execution_payload)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This is the beacon chain specification of the enshrined proposer builder separation feature. 
+
+*Note:* This specification is built upon [Deneb](../../deneb/beacon-chain.md) and is under active development.
+
+This feature adds new staked consensus participants called *Builders* and new honest validators duties called *payload timeliness attestations*. The slot is divided in **four** intervals as opposed to the current three. Honest validators gather *signed bids* from builders and submit their consensus blocks (a `SigneddBeaconBlock`) at the beginning of the slot. At the start of the second interval, honest validators submit attestations just as they do previous to this feature). At the  start of the third interval, aggregators aggregate these attestations (exactly as before this feature) and the honest builder reveals the full payload. At the start of the fourth interval, some honest validators selected to be members of the new **Payload Timeliness Committee** attest to the presence of the builder's payload.
```

Comment:
```suggestion
This feature adds new staked consensus participants called *Builders* and new honest validators duties called *payload timeliness attestations*. The slot is divided in **four** intervals as opposed to the current three. Honest validators gather *signed bids* from builders and submit their consensus blocks (a `SignedBeaconBlock`) at the beginning of the slot. At the start of the second interval, honest validators submit attestations just as they do previous to this feature). At the  start of the third interval, aggregators aggregate these attestations (exactly as before this feature) and the honest builder reveals the full payload. At the start of the fourth interval, some honest validators selected to be members of the new **Payload Timeliness Committee** attest to the presence of the builder's payload.
```

Reply:
had to read this like 10 times to find it